apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/CcwRifLoadJobIT.java

```diff
@@ -507,6 +518,41 @@ public void skipDataSetTestForFutureManifestDate() throws Exception {
     }
   }
 
+  /**
+   * Tests {@link CcwRifLoadJob} when run twice; once against a bucket that uses synthetic data but
+   * will not have a {@link SyntheaEndStateProperties} object as part of the manifest, and a second
+   * time (same bucket) where the manifest includes a {@link SyntheaEndStateProperties} that is
+   * invalid per its XML Schema Definition (XSD).
+   *
+   * @throws Exception (exceptions indicate test failure)
+   */
+  @Test
+  public void skipDataSetTestForSyntheaPreValidationFailed() throws Exception {
+    validateLoadAtLocations(
+        CcwRifLoadJob.S3_PREFIX_PENDING_SYNTHETIC_DATA_SETS,
+        CcwRifLoadJob.S3_PREFIX_COMPLETED_SYNTHETIC_DATA_SETS,
+        false);
+
+    // job ran and now we should have some Synthea data in the db; we'll re-run
+    // the same dataset but this time specifying Synthea end-state information.
+    Exception exception =
```

Comment:
all tests for the PreValidation were moved to RifLoaderIT since all we really care about is vetting data and that class is more suitable spot for the tests.